doc/source/contents.rst

```diff
@@ -0,0 +1,31 @@
+.. _contents:
+
+Documentation Table of Contents 
+===============================
+
+.. toctree::
+    :includehidden:
+    :titlesonly:
+    :maxdepth: 5
+
+    Home <index>
+    Table of Contents <self>
+    Downloads and Links <downloads_links>
+    Tutorials <tutorials>
+    Community Example Library <library>
+    General <general>
+    Language <language>
+    Mathematics <math>
+    Commands <commands>
+    Structures <structures>
+    Addons <addons>
+    Contribute <contribute>
+    Changes <changes>
+    About <about>
+
+Indices and tables
+==================
+
+* :ref:`genindex`
+* :ref:`modindex`
```

Comment:
How should this work? My Sphinx did not generate a `modindex` ref and as a result the link doesn't work.